runtime/src/main/java/io/novafoundation/nova/runtime/multiNetwork/chain/model/Chain.kt

```diff
@@ -31,6 +31,7 @@ data class Chain(
     val hasSubstrateRuntime: Boolean,
     val hasCrowdloans: Boolean,
     val governance: List<Governance>,
+    val swapSupporting: Boolean,
```

Comment:
Do we use it already? We should adapt `SwapService` to take this flag into account

Reply:
Yes we already use it in `createExchanges`
<img width="455" alt="image" src="https://github.com/novasamatech/nova-wallet-android/assets/107959809/30a4d1cb-0fdf-4a9b-8aa6-ec120ca952f2">